includes/widgets/button.php

```diff
@@ -444,15 +444,7 @@ protected function render() {
 
 		if ( ! empty( $settings['link']['url'] ) ) {
 			$this->add_render_attribute( 'button', 'href', $settings['link']['url'] );
```

Comment:
seems Wrong, as you don't need this anymore

Reply:
Removed